code/modules/projectiles/guns/special.dm

```diff
@@ -0,0 +1,112 @@
+/obj/item/weapon/gun/portalgun	//-by Deity Link
+	name = "\improper Portal Gun"
+	desc = "There's a hole in the sky... through which I can fly."
+	icon = 'icons/obj/gun_experimental.dmi'
+	icon_state = "portalgun0"
+	item_state = "portalgun0"
+	slot_flags = SLOT_BELT
+	origin_tech = "materials=7;bluespace=6;magnets=5"
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/guns_experimental.dmi', "right_hand" = 'icons/mob/in-hand/right/guns_experimental.dmi')
+	recoil = 0
+	flags = FPRINT
+	w_class = 3.0
+	fire_delay = 0
+	fire_sound = 'sound/weapons/portalgun_blue.ogg'
+	var/setting = 0	//0 = Blue, 1 = Red.
+	var/obj/effect/portal/blue_portal = null
+	var/obj/effect/portal/red_portal = null
+
+/obj/item/weapon/gun/portalgun/examine(mob/user)
+	..()
+	switch(setting)
+		if(0)
+			user << "It's current setting is <font color='#0066FF'>blue</font>."
```

Comment:
@DeityLink we also had a policy against things deprecated in HTML 5, `<font>` is one of them.

If you need to set the colour: `<span style="color: [COLOR];"></span>`